Code/Tools/ProjectManager/Source/ProjectInfo.cpp

```diff
@@ -98,4 +98,16 @@ namespace O3DE::ProjectManager
             return m_projectName;
         }
     }
+
+    void ProjectInfo::SetProjectDisplayName(const QString& displayName)
+    {
+        if (displayName.size() > 25)
+        {
+            m_displayName = displayName.left(22) + "...";
```

Comment:
Please use a const variable for the max length value and then use `maxLength-3` instead of `22`. It will be easier to change the length in the future.